src/lando/main/models/landing_job.py

```diff
@@ -347,28 +347,14 @@ def transition_status(
 
         self.save()
 
-    def serialize(self) -> dict[str, Any]:
-        """Return a JSON compatible dictionary."""
-        return {
-            "id": self.id,
-            "status": self.status,
-            "landing_path": self.serialized_landing_path,
-            "error_breakdown": self.error_breakdown,
-            "details": (
-                self.error or self.landed_commit_id
-                if self.status in (LandingJobStatus.FAILED, LandingJobStatus.CANCELLED)
-                else self.landed_commit_id or self.error
-            ),
-            "requester_email": self.requester_email,
-            "tree": self.repository_name,
-            "repository_url": self.repository_url,
-            "created_at": (
-                self.created_at.astimezone(datetime.timezone.utc).isoformat()
-            ),
-            "updated_at": (
-                self.updated_at.astimezone(datetime.timezone.utc).isoformat()
-            ),
-        }
+    @property
+    def legacy_details(self) -> str:
+        """Return a string of the landed commit id or the error details."""
+        return (
+            self.error or self.landed_commit_id
+            if self.status in (LandingJobStatus.FAILED, LandingJobStatus.CANCELLED)
+            else self.landed_commit_id or self.error
```

Comment:
Would there be a way to simplify this expression? The duplication of terms is breaking my brain 😅

Reply:
I will take a stab at simplifying it.

Reply:
I went back to check, this code is very old. I don't think we need to still handle the case of showing a "landed commit ID" when a job is failed or deferred, since that should not ever happen (`job.landed_commit_id` is only set when a job is marked as `LANDED`).